content/kubeone/main/known-issues/_index.en.md

```diff
@@ -1,138 +1,118 @@
 +++
 title = "Known Issues"
-date = 2022-08-29T09:00:00+02:00
+date = 2023-02-21T09:00:00+02:00
 description = "Known Issues in Kubermatic KubeOne"
 weight = 99
 +++
 
 This page documents the list of known issues in Kubermatic KubeOne along with
 possible workarounds and recommendations.
 
-This list applies to KubeOne 1.5 releases. For earlier releases, please
-consult the [appropriate changelog][changelogs].
+This list applies to KubeOne 1.6 releases. For KubeOne 1.5, please consider
+the [v1.5 version of this document][known-issues-1.5]. For earlier releases,
+please consult the [appropriate changelog][changelogs].
 
-[changelogs]: https://github.com/kubermatic/kubeone/tree/mai/CHANGELOG
+[known-issues-1.5]: {{< ref "../../v1.5/known-issues" >}}
+[changelogs]: https://github.com/kubermatic/kubeone/tree/main/CHANGELOG
 
-## Pod connectivity is broken for Calico VXLAN clusters
+## `node-role.kubernetes.io/master` taint not removed on upgrade when using KubeOne 1.6.0-rc.1
 
 |              |                                                   |
 |--------------|---------------------------------------------------|
-| Status       | Being Investigated                                |
-| Severity     | High for clusters using Calico VXLAN addon        |
-| GitHub issue | https://github.com/kubermatic/kubeone/issues/2192 |
-
-### Description
+| Status       | Fixed in KubeOne 1.6.0-rc.2                       |
+| Severity     | Critical                                          |
+| GitHub issue | https://github.com/kubermatic/kubeone/pull/2688   |
 
-Clusters running Calico VXLAN might not be able to reach ClusterIP Services
-from a node where the pod is running.
-
-### Recommendation
+Users who:
 
-**We do NOT recommend upgrading to KubeOne 1.5 at this time if you're using
-Calico VXLAN. Follow the linked GitHub issue and this page for updates.**
+- used **KubeOne 1.6.0-rc.1** or built KubeOne manually on
```

Comment:
Probably it's better to put accent on the fact that the fix is available after _1.6.0-rc.1 or later_, with emphasis on later (having a mind a release version), otherwise it looks like as if we are recommending an RC pre-release version over the release (yes, at this point it's future release, but still).

Reply:
OK, scratch that, I've read it wrongly